File: pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;


import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

public interface BrandService {
    /**
     * 分页查询
     * @param pageNum
     * @param pageSize
     * @param brand
     * @return
     */
    PageResult findPage(Integer pageNum, Integer pageSize, TbBrand brand);

    /**
     * 新建
     * @param brand
     */
    void save(TbBrand brand);

    /**
     * 查询单个
     * @param id
     * @return
     */
    TbBrand findOne(Long id);

    /**
     * 修改信息
     * @param brand
     */
    void update(TbBrand brand);

    /**
     * 删除选中
     * @param ids
     */
    void delete(Long[] ids);


}

File: pinyougou_manager_web/src/main/webapp/js/service/brandService.js
//brandService
app.service("brandService",function ($http) {
    //新建
    this.add = function (entity) {
        return $http.post("../brand/add.do", entity);
    };

    //更新
    this.update=function (entity) {
        return $http.post("../brand/update.do",entity);
    };
    //查询修改项
    this.findOne = function (id) {
        return $http.get("../brand/findOne.do?id="+ id);
    };
    //删除
    this.delSelected =function (ids) {
        return $http.get("../brand/delete.do?ids=" +ids);
    };
    //查询
    this.search =function (page,rows,searchEntity) {
        return $http.post('../brand/search.do?page=' + page + '&rows=' + rows, searchEntity);
    };
    //品牌下拉
    this.selectOptionList=function () {
        return $http.get("../brand/selectOptionList.do");
    }

});